src/cnxml/index.js

```diff
@@ -30,7 +30,7 @@ const serializer = new Html({
 export default {
     deserialize: (...args) => serializer.deserialize(...args),
 
-    serialize(value) {
+    serialize(value) {        
```

Comment:
In the future strip such changes before sending a PR.

Reply:
How can I do this?